docs/whatsnew-1.8.rst

```diff
@@ -0,0 +1,40 @@
+What's New in Pyramid 1.8
```

Comment:
Please don't add a whatsnew document. We will do it prior to release. Your changes are good I just think it's easier to create that file from scratch just before a major release.